src/app/berkeley_migration_verifier/sql.ml

```diff
@@ -296,9 +296,12 @@ module Berkeley = struct
   let dump_account_accessed_to_csv_query ~output_file =
     dump_sql_to_csv output_file
       ~sql:
-        "SELECT account_identifier_id AS id, block_id \n\
-        \            FROM accounts_accessed \n\
-        \            ORDER BY block_id, id \n"
+        {sql| SELECT account_identifier_id AS id, block_id
+                 FROM accounts_accessed
+                 JOIN blocks ON block_id = blocks.id
+                 WHERE height <> 1
```

Comment:
Why this SQL was changed in this PR?

Reply:
This should be fixed in a prior PR: https://github.com/MinaProtocol/mina/pull/15313
It's because with the addition of genesis accounts in `accounts_accessed`, then this check should ignore those genesis accounts.
I was using `berkeley_migration_verifier` to check my PR, so I had this change here.

Reply:
Okay, so it's just a fixup for a previous PR